src/datacollector/Resources/xlf/Resources.cs.xlf

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
+  <file datatype="xml" source-language="en" target-language="cs" original="../Resources.resx" build-num="-191745455">
+    <header>
+      <count-group name="BlackBox wordcount">
+        <count count-type="x-wordCount">27</count>
+        <count count-type="x-adjWordCount">22.95</count>
+        <count count-type="x-curAdjWordCount">22.95</count>
+        <count count-type="x-repeatAdjWordCount">0</count>
+        <count count-type="x-termWordCount">0</count>
+      </count-group>
+      <count-group name="BlackBox wordcount analysis">
+        <count count-type="x-match102%">0</count>
+        <count count-type="x-match101%">0</count>
+        <count count-type="x-match100%">0</count>
+        <count count-type="x-match99-90%">0</count>
+        <count count-type="x-match89-75%">0</count>
+        <count count-type="x-mt">27</count>
+        <count count-type="repetition">0</count>
+      </count-group>
+    </header>
+    <body>
+      <trans-unit id="DataCollectorFailedToConnetToVSTestConsole">
+        <source>Datacollector process failed to connect to vstest.console in {0} milliseconds, Set enviroment variable {1} to increase timeout.</source>
+        <target state="new">Datacollector process failed to connect to vstest.console in {0} milliseconds, Set enviroment variable {1} to increase timeout.</target>
```

Comment:
>Datacollector process failed to connect to vstest.console in {0} milliseconds [](start = 28, length = 77)

Datacollector process failed to connect to vstest.console in {0} milliseconds. Please set environment variable {1} to increase the connection timeout.

Reply:
@singhsarab @mayankbansal018 Agreed. Done.